models/chronic_conditions/final/chronic_conditions__tuva_chronic_conditions_long.sql

```diff
@@ -1,5 +1,5 @@
 {{ config(
-     enabled = var('tuva_chronic_conditions_enabled',var('tuva_marts_enabled',True))
+     enabled = var('tuva_chronic_conditions_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))
```

Comment:
It looks like Tuva Chronic Conditions runs off of both claims and clinical since the stage refs core__condition and core__patient. Should we allow this to be enabled by either clinical or claims?